### ISSUE TITLE
UI Theme Refactoring

### ISSUE THREAD

**Issue by kryptot7** [zrispo/wick#819]
A couple weeks ago, I posted a screenshot of a beta 0.14.1 running modified CSS files:
![image](https://user-images.githubusercontent.com/30781325/37786676-9f8926d4-2dd3-11e8-935d-7cec8f6ce102.png)
Here are the CSS files themselves! ^^ They're a bit out of date now, but I adjusted many of the colors to refer to CSS variables (--background-light-1, etc.) instead of hex values. (I missed a few, but those aren't as obvious.) I personally don't think Wick needs a theme loader at this point, but this would be a great step in enabling that feature.
This also means that if you don't like the brownish tint I applied, you can easily change them to more fitting shades by adjusting the variables. I really like the green accent instead of blue, though. 😄 That and the font make it much more consistent with the website.
[css_files.zip](https://github.com/zrispo/wick-editor/files/1838561/css_files.zip)

**Comment by zrispo** [comment]
Dumb question, but how do I get the css files into wick (just to try the theme)? Can you do it through the dev console?

(Eventually i'll make a proper theme loader but that will happen later)

**Comment by kryptot7** [comment]
I just downloaded the repository as a .zip and edited the source files, so you have to download/clone the version from March 6th (which is when I downloaded it) and then replace all the .css's in /styles. 🙂

**Comment by zrispo** [comment]
Oh i see!

Do you know how to fork projects on github? It makes it really easy to share code + merge changes of other people. Let me know if you need any help, I'd be happy to show you how to make forks and pull requests - then we can get you started contributing to the codebase!!

**Comment by kryptot7** [comment]
No, but I've downloaded SourceTree, and I'll be experimenting with it later 😃
